modular_skyrat/modules/opposing_force/code/opposing_force_equipment.dm

```diff
@@ -0,0 +1,572 @@
+/datum/opposing_force_equipment/gun/m16m
+	item_type = /obj/item/gun/ballistic/automatic/assault_rifle/m16/modern
+
+/datum/opposing_force_equipment/gun/m16patriot
+	item_type = /obj/item/gun/ballistic/automatic/assault_rifle/m16/modern/v2
+
+/datum/opposing_force_equipment/gun/akm
+	item_type = /obj/item/gun/ballistic/automatic/assault_rifle/akm
+
+/datum/opposing_force_equipment/gun/akmm
+	item_type = /obj/item/gun/ballistic/automatic/assault_rifle/akm/modern
+
+/datum/opposing_force_equipment/gun/mp40
+	item_type = /obj/item/gun/ballistic/automatic/submachine_gun/mp40
+
+/datum/opposing_force_equipment/gun/mp40m/modern
+	item_type = /obj/item/gun/ballistic/automatic/submachine_gun/mp40
+
+/datum/opposing_force_equipment/gun/pps
+	item_type = /obj/item/gun/ballistic/automatic/submachine_gun/pps
+	description = "A very cheap, barely reliable reproduction of a personal defense weapon based on the original Soviet model. Not nearly as infamous as the Mosin. \ Compact enough to fit in your backpack!"
+
+/datum/opposing_force_equipment/gun/ppsh
+	item_type = /obj/item/gun/ballistic/automatic/submachine_gun/ppsh
+
+/datum/opposing_force_equipment/gun/ppshm
+	item_type = /obj/item/gun/ballistic/automatic/submachine_gun/ppsh/modern
+
+/datum/opposing_force_equipment/gun/stg
+	item_type = /obj/item/gun/ballistic/automatic/assault_rifle/stg
+
+/datum/opposing_force_equipment/gun/stgm
+	item_type = /obj/item/gun/ballistic/automatic/assault_rifle/stg/modern
+
+/datum/opposing_force_equipment/gun/fg42
+	item_type = /obj/item/gun/ballistic/automatic/battle_rifle/fg42
+
+/datum/opposing_force_equipment/gun/fg42m
+	item_type = /obj/item/gun/ballistic/automatic/battle_rifle/fg42/modern
+
+/datum/opposing_force_equipment/gun/uzi
+	item_type = /obj/item/gun/ballistic/automatic/mini_uzi
+	description = "The uzi nine millimeter, a timeless submachinegun for a warrior out of time."
+
+/datum/opposing_force_equipment/gun/mg34
+	item_type = /obj/item/gun/ballistic/automatic/mg34
+
+/datum/opposing_force_equipment/gun/cfa
+	item_type = /obj/item/gun/ballistic/automatic/cfa_rifle
+
+//oldarmory shotguns, now rebalanced for opfor store
+/datum/opposing_force_equipment/gun/m23
+	item_type = /obj/item/gun/ballistic/shotgun/m23
+	description = "An eight-round pump-action shotgun found in an old station. Comes loaded with beanbag shells but can take any 12 gauge load."
+
+/datum/opposing_force_equipment/gun/as2
+	item_type = /obj/item/gun/ballistic/shotgun/automatic/as2
+	description = "A compact version of the combat shotgun. Comes with a four-round tube and can be silenced as well as holstered to your belt or vest. \
+		Comes loaded with slugs but can take any 12 gauge load."
+
+/datum/opposing_force_equipment/gun/sas14
+	item_type = /obj/item/gun/ballistic/shotgun/sas14
+	description = "A semi-automatic, magazine-fed shotgun chambered in 14 gauge. Standard mags can take five rounds. \
+		Despite the lower damage of 14 gauge, it can load the highly praised taser shots."
+
```

Comment:
The line breaks should be normalized to a certain line rather than being thrown around randomly, if you're going to do them.